src/lib.rs

```diff
@@ -279,3 +279,22 @@ impl<T: MutEventSubscriber + ?Sized> MutEventSubscriber for Box<T> {
         self.deref_mut().init(ops);
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn test_subscriber_id_derives() {
```

Comment:
Is there any reason for checking the derives? We considered this to be coverage tests, and we avoided them in the past. I am wondering if there is something particular about the `SubscriberId` that we want to validate, or we could remove them.